2nd phase of deprecation of "x ident" in constr notation (now an error) (#14637)
coq/coq: doc/changelog/03-notations/14637-master+notation-type-ident-deprecation-error-phase-8.14.rst

- **Added:**

>   New notation ``strict ident`` in :n:`@explicit_subentry` to ease the transition from
>   the poorly named ``ident`` to ``name``.  Currently (except in custom
>   notations), ``ident`` accepts :n:`@name`, that is, either :n:`@ident`

Reviewer: ```suggestion
  New notation ``strict ident`` in :n:`@explicit_subentry` that only accepts
  :n:`@ident`.  This is meant to ease the transition from
  the inaccurately named ``ident`` to ``name``.  Currently (except in custom
```

Following @silene's comment.
